docs/faq.rst

```diff
@@ -0,0 +1,21 @@
+Frequently-Asked Questions
+==========================
+
+When I try to run my Python file from the shell or Brickman,
+it throws errors or exits immediately
+-----
+
+This may occur if your file includes Windows-style line endings, which are often
+inserted by editors on Windows. To resolve this issue, open an SSH session and
+run the following command, replacing ``<file>`` with the name of the Python file
+you're using:
+
+.. code::
+
+    sed -i 's/\r//g' <file>
```

Comment:
It still exists but is not installed by default, so I decided to go with the built-in option.